KsApi/queries/FetchAddOnsQuery.graphql

```diff
@@ -1,4 +1,4 @@
-query FetchAddOns($projectSlug: String!, $shippingEnabled: Boolean!, $locationId: ID) {
+query FetchAddOns($projectSlug: String!, $shippingEnabled: Boolean!, $locationId: ID, $withStoredCards: Boolean!) {
```

Comment:
I might be missing something here but I see the parameter declaration here but where is it being passed into the fragment for the declarative?

Reply:
yea I was curious about that too. Having done some testing with GraphiQL, I found queries like this just pass down the parameter into fragments of fragments that use it. I suppose when everything gets compiled it's just one long string that is like one big query and all the parameters get referenced somewhere.

Try this out in GraphiQL to get an idea of not needing to pass every fragment a copy of the parameter:
```
query FetchUser($withStoredCards: Boolean = true) {
  me {
    ...UserFragment
  }
}

fragment UserFragment on User {
  chosenCurrency
  email
  hasPassword
  id
  imageUrl: imageUrl(blur: false, width: 1024)
  isAppleConnected
  isCreator
  isDeliverable
  isEmailVerified
  name
  uid
  storedCards @include(if: $withStoredCards) {
    __typename
    ...UserStoredCardsFragment
  }
}

fragment UserStoredCardsFragment on UserCreditCardTypeConnection {
    nodes {
      __typename
      expirationDate
      id
      lastFour
      type
    }
    totalCount
}
```

Also taking this parameter out causes compile errors even for seemingly unrelated queries like `FetchAddOns`.

Reply:
Ah yeah looks like it's doing that for us. Cool stuff.